code/modules/assembly/flash.dm

```diff
@@ -301,6 +301,17 @@
 	inhand_icon_state = "nullrod"
 	lefthand_file = 'icons/mob/inhands/weapons/melee_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/melee_righthand.dmi'
+	var/implant_message = "You don't remember anything between the start of the shift and now"
+
+/obj/item/assembly/flash/memorizer/attack(mob/living/M, mob/user)
```

Comment:
Avoid one-letter vars, and use interact_with_atom instead of overriding attack()